### ISSUE TITLE
make sonerezh ignore apetags?

### ISSUE THREAD

**Issue by pacija** [gs11/sonerezh#2]
Hi,

I've spent last few days trying to find out why some of my tracks get imported with strange tags, even if I completely strip IDv1 and IDv2.4 tags. It appears this is due to the presence of apetag (I found this by checking tracks directly with getID3 demo).

Could sonerezh be modified to ignore apetags and use ID tags only?

Thank you in advance.

**Comment by gs11** [comment]
Not familiar enough with getID3 to say but I'd figure there's some kind of priority in the tag reading.
The issue with ignoring ape tags is that there may be users who rely solely on them.

**Comment by pacija** [comment]
Perhaps tweak priority then? Or make it configurable? I'd say ID3v2.X is the standard that majority of people use. I'm listening to mp3s for almost a decade now, and up until recently I haven't heard about apetags. 99% of my audio library has no apetags. None of my audio players reads and displays apetags.

Sonerezh shouldn't give priority to apetags instead of ID3v tags.

**Comment by gs11** [comment]
I'll look into what can be done through getID3 parameters, if priority can be used.

**Comment by MightyCreak** [comment]
Some people might love APE tags (which I didn't know of before either) and want them to be read in priority over standard ID3 tags.

Personally, and considering the resources of Sonerezh, I'm all for the users to take care of their music library instead of tweaking Sonerezh to by-pass behaviours that a few don't like.

If we would like to fix this problem, the first solution to come in mind is to add just one tiny little checkbox in the Sonerezh settings that would enable/disable APE tag reading, but that is ultimately bad because it creates yet another code path that needs to be tested, each time, to verify there's no regression. And if we do that for each and every different users tastes, we'll end up with a shitload of options in the settings, that 99% of the users will find mostly useless.

TL;DR: why not simply remove or change the APE tags from your 1% audio files?

**Comment by gs11** [comment]
Found no way of getting getID3 to use tag prioritization other than default. I fully agree with @MightyCreak here and also have a hard time ratifying a complex solution for a very small use case.
